### DISCUSSION TITLE
Support multiple temp disks in cascade

### DISCUSSION THREAD

**Discussion by maxpiva** [Chia-Network/chia-blockchain#5211] [Ideas]
As the title says, support them in cascade. When there is not enough space in the first one, check the second one, and so on.....

**Comment by SackofFluor** [comment]
Isnt that actually implemented by the secondary temp drive? How many backup-Temps do you need 😅😅

**Reply by maxpiva** [reply]
From my understanding, tmp2 is only used for phase 3 compression. Not sure also, if it's used as a cascade in that stage, or force used.
